packages/itwinui-react/src/index.ts

```diff
@@ -111,6 +111,8 @@ export { NotificationMarker } from './core/NotificationMarker/NotificationMarker
 
 export { Overlay } from './core/Overlay/Overlay.js';
 
+export { Panels } from './core/Panels/Panels.js';
```

Comment:
Do we want to release this component as "unstable" to allow for API changes based on feedback?

Just the export name can be changed:

```suggestion
export { Panels as unstable_Panels } from './core/Panels/Panels.js';
```

And then it would be used as:

```jsx
import { unstable_Panels as Panels } from "@itwin/itwinui-react";
```

Reply:
CI is failing. All the imports will need to be updated.

Also, we will need docs on day one when we release this. You can start working on it while I'm taking a final look at the PR.